src/app/config/config.c

```diff
@@ -5734,7 +5734,7 @@ options_init_logs(const or_options_t *old_options, or_options_t *options,
         goto cleanup;
       }
 
-      if (!strcasecmp(smartlist_get(elts, 0), "android")) {
+      if (!strcasecmp(smartlist_get(elts, 0), "logcat")) {
```

Comment:
I'm happy to accept that "logcat" is a better name here, but it isn't okay to break backward compatibility with everybody who was using the old name.  If we want to make this change, we should keep "android" as a deprecated synonym.

Additionally this should probably be documented someplace.

Reply:
I could find no evidence that the "android" name was in use in a shipped app.  I looked in Orbot, Briar, and Tor Browser.  Seemed cleaner to just drop the old name and move on.  Do you know somewhere where it is used?

Reply:
I started writing the docs for this, and then did some tests.  I've changed my mind.  I think the "android" option should just be removed.  syslog works well on Android.
https://trac.torproject.org/projects/tor/ticket/32181

Since the `android` log option was never documented, and it wasn't used by the well known Tor apps.  It always seemed to me to be a private/alpha API, so better to just remove the cruft, perhaps there could be an error if people use it?  I'm going to update the docs in this pull request, and the changes file to point Android users to syslog.

Reply:
I'll remove this change from this pull request.

Reply:
sounds good.